requirements-runners.txt

```diff
@@ -1,2 +1,6 @@
 ARS_Test_Runner==0.0.2
 ui-test-runner==0.0.1
+
+# Dependency for the ui-test-runner
+# not automatically pulled in
+jq==1.6.0
```

Comment:
This is my bad. This can be removed and the `ui-test-runner` version should be bumped to 0.0.2 to include jq as a sub-dependency.